test/runtests.jl

```diff
@@ -50,19 +52,8 @@ end == 42
 @test_throws ErrorException @suppress begin
     println("This string doesn't get printed!")
     warn("This warning is ignored.")
-    error("Remember that errors are still printed!")
+    error("Remember that errors are still printed!")  # they aren't; @test_throws suppresses them
```

Comment:
should this string be changed then to something like `"errors would normally get printed but are caught here by @test_throws"` ?

Reply:
Changed